src/reducers/gists.js

```diff
@@ -0,0 +1,16 @@
+import * as actions from '../actions/gists';
+
+export const initialState = [];
+
+const gists = (state = initialState, action) => {
+  switch (action.type) {
+  case actions.SET_GISTS: {
+    return action.payload;
+  }
+  default: {
+    return state;
+  }
+  }
+};
+
+export default gists;
```

Comment:
Can this not go in the `app` reducer? I ask because adding more reducers means cleaning up the app (like when the user clicks the title in the AppBar) becomes harder and less efficient to do because we have to remember to dispatch additional actions, and the fact that we're dispatching more actions has an impact on the performance (even if slight). Plus, there's only one action you're handling in here.